modules/API/pages/built-in-endpoints.adoc

```diff
@@ -1757,6 +1757,41 @@ For example, the following payload is used to upsert two `User` vertices with an
 }
 ----
 
+==== Composite keys
+
+If your vertex has composite keys, separate the attributes that make up the composite key with comma (`,`) in the same order as they are defined in the schema.
+
+For example, suppose we have the following vertex definition:
+
+[.wrap,gsql]
+----
+CREATE VERTEX compositePerson(id UINT, name STRING, age UINT, primary key (name, id))
+----
+
+The following requests upserts two vertices with the defined composite key:
+
+[.wrap,console]
+----
+curl -X POST "localhost:9000/graph/person_movie" -d '
+{
+  "vertices": {
+    "compositePerson":{
+      "Bob,123":{
+        "name":{"value":"Bob"},
+        "id":{"value":123},
+        "age":{"value":25}
+      },
+      "Tom,456":{
+        "name":{"value":"Tom"},
+        "id":{"value":456},
+        "age":{"value":47}
+      }
+    }
+  }
+}
```

Comment:
can you also add the "edges" into this example? like:
```
"edges":{
    "compositePerson":{
      "Bob,123":{
        "compositeRoles":{
          "compositeMovie":{
            "Harry Potter,1990,1337":{
              "role":{
                "value":"Wizard"
              }
            }
          }
        }
      }
    }
  }
```

Reply:
added

Reply:
Please also add the create vertex/edge for easier understanding.

create vertex compositeMovie (id uint, title string, country string, year uint,  primary key (title,year,id))
create directed edge compositeRoles (from compositePerson,to compositeMovie, role string)

Reply:
added